code/datums/jobs/job/shipside.dm

```diff
@@ -67,11 +67,11 @@ Godspeed, captain! And remember, you are not above the law."})
 	if(!playtime_mins || playtime_mins < 1 )
 		return
 	switch(playtime_mins)
-		if(0 to 500)
+		if(0 to 1500) // starting
 			new_human.wear_id.paygrade = "O6"
-		if(501 to 10000)
+		if(1501 to 7500) // 25hrs
 			new_human.wear_id.paygrade = "O7"
-		if(10001 to INFINITY)
+		if(7501 to INFINITY) //125 hrs
```

Comment:
This is pretty big jumps are you sure about this

Reply:
All command roles meant to have the max rank at 125 hrs, it's only being reduced by about 42hrs, which isn't that lot when taking the overall hours into account.
If someone spends over 125hrs playing Cap I doubt that an extra 40-ish hrs would make much of a difference.

Reply:
Mhmm sure I guess